frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -52,23 +54,19 @@ export function Experiment(): JSX.Element {
 
     const conversionRate = conversionMetrics.totalRate * 100
     const entrants = results?.[0]?.count
+    const sampleSize = recommendedSampleSize(conversionRate)
+    const runningTime = expectedRunningTime(entrants, sampleSize)
 
     return (
         <>
-            {experimentId === 'new' || !experimentData?.start_date ? (
+            {experimentId === 'new' ? (
```

Comment:
Why don't we want a draft experiment to open up from where the user left off on it? (in its edit mode)

Reply:
I actually got rid of the save as draft button as well (since same semantics as save):

1. We can't save partial information anyways - save works only when all required information is filled in. And at that time, it's literally a confirmation screen.

2. The mocks seem to have a WIP confirmation screen with edit & launch functionality:

![image](https://user-images.githubusercontent.com/7115141/146393188-f3649b45-64ff-4084-93b3-de86ea059661.png)

(So, when you click edit, you go back to this view, otherwise stay on confirmation screen)